packages/components/src/components/as-histogram-widget/as-histogram-widget.e2e.ts

```diff
@@ -104,16 +104,57 @@ describe('as-histogram-widget', () => {
       expect(ticksValues).toEqual(['-0.12', '0.88', '1.9', '2.9', '3.9']);
     });
 
-    it('should format X axis using SI prefixes when range is lower than 1', async () => {
+    it(
+      'should format X axis using up to 3 decimals when range is lower than 1 using values with precision lower than 3',
```

Comment:
it says 'using up to 3 decimals' but the assert displays just up to 2. Should we rename it? add more elements to histogram to get an example with that 3 ?